addons/website/static/src/snippets/s_image_gallery/options.js

```diff
@@ -310,9 +309,15 @@ snippetOptions.registry.gallery = snippetOptions.SnippetOptionWidget.extend({
         this._super(...arguments);
         if (name === 'image_removed') {
             data.$image.remove(); // Force the removal of the image before reset
-            await this.mode('reset', this.getMode());
+            if(this._getImages().length === 0){
```

Comment:
New basic linter errors...

Reply:
I created a point in our pad for the linting process. We have yet to figure out how to lint ts and js files separately with a different set of rules since both use eslint.